cmd/controller-manager/main.go

```diff
@@ -54,6 +55,11 @@ func main() {
 		}
 	}
 
+	err = rspcontroller.StartReplicaSchedulingPreferenceController(config, config, config, stopChan, true)
```

Comment:
Up till now the only functionality of v2 has been propagation.  Adding scheduling suggests a requirement for a configuration mechanism that would allow scheduling to be optionally enabled.

Reply:
Configuration seems reasonable to me. I can take that up once I complete relevant tasks needed to make the scheduling API usable ([listed here](https://github.com/kubernetes-sigs/federation-v2/pull/46#issuecomment-384913451)).